mathics/builtin/assignments/internals.py

```diff
@@ -174,7 +174,7 @@ def unroll_patterns(lhs, rhs, evaluation):
 
 def unroll_conditions(lhs):
     condition = None
-    if type(lhs) is Symbol:
```

Comment:
I believe it is for exactly this kind of change that Python lint checkers tell you not to use 
this idom.